src/applications/simple-forms/40-0247/containers/IntroductionPage.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import { focusElement } from 'platform/utilities/ui';
+import OMBInfo from '@department-of-veterans-affairs/component-library/OMBInfo';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+
+class IntroductionPage extends React.Component {
```

Comment:
same question about `/shared/components/IntroductionPageView`. I know that yeoman gives you these pages instead but maybe we can switch to using the shared components in the future

Reply:
yes, will try that out during Build-frontend step.  Thanks!